Enterprise-CMCS/eAPD
# 4474 Executive Summary update for MMIS

Resolves #4474 

### Description
Updates MMIS Executive Summary - Program Budget Tables to be split out by DDI and M&O

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4636&view=activity

### Significant changes or possible side effects
None noted

### Automated test cases written
Jest test for verifying rendering with valid budget and budget with 0 totals

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| an MMIS APD | viewing the Program Budget tables with all $0 totals | tables do not render | jest |
| an MMIS APD | viewing the Program Budget tables with a valid sample budget | tables render correctly | jest |

### Steps to manually verify this change
1. Login to a HITECH APD and verify no changes to existing Program Budget tables (in executive summary)
2. Login to a MMIS APD and verify new tables are present in Executive Summary
3. Verify tables match design spec

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
